src/dnx.common/include/app_main.h

```diff
@@ -21,3 +21,13 @@ typedef struct CALL_APPLICATION_MAIN_DATA
 #endif
     *FnCallApplicationMain)(
     PCALL_APPLICATION_MAIN_DATA pCallApplicationMainData);
+
+struct bootstrapper_context
```

Comment:
So maybe I'm being paranoid, but I'd really like to try and keep these in sync. Could we put this and the matching C# file together somehow? Or at the very least put a big giant comment indicating that these two MUST be kept in sync.

If I had my way and the necessary tools existed, I'd have one of the two code-genned from the other ;). Maybe Roslyn could be used to generate the unmanaged struct? Anyway, that can wait.